joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -417,12 +123,12 @@ TEST_F(TestTrajectoryController, configuration) {
 //   executor.spin_once();
```

Comment:
Why are those tests commented out?

Reply:
Not sure, the went commenteed on 93d152b and noone has bothered to check.

Reply:
https://github.com/ros-controls/ros2_controllers/issues/118 lovely title for an issue